test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -821,7 +821,7 @@ def check_infiltration(prototypes, prototypes_base)
   # Expected outcome depends on prototype name and 'mod' variation defined with
   #
   # @param prototypes_base [Hash] Baseline prototypes
-  def check_hvac_type(prototypes_base)
+  def check_hvac(prototypes_base)
```

Comment:
Changed the name of the method as I included the fan power calculation verification as part of it.